modules/readme/src/main/tut/README.md

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Did you remove this first empty line? Remember it's needed if you want from sbt-org-policies to replaces badges automatically.